requirements.txt

```diff
@@ -1,6 +1,6 @@
+-r Rasa_Bot/requirements.txt
```

Comment:
Cool I didn't know about this! We should actually also do this for the other 'sub' applicaitons. Can you create a new issue for that?